service/src/main/java/bio/terra/pipelines/db/entities/PipelineInputs.java

```diff
@@ -13,7 +13,7 @@
 @Setter
 @NoArgsConstructor
 @Table(name = "pipeline_inputs")
-public class PipelineInput {
+public class PipelineInputs {
```

Comment:
oh i think the interface with the API should be PipelineInputs but I think this should still be `PipelineInput`.  Same for the `PipelineOutput` entity

Reply:
hm, even though each instance of this entity contains the same information as the API interface, i.e. multiple outputs?